src/Step/Build/CopyPubStatic.php

```diff
@@ -54,16 +63,14 @@ public function execute(): void
     {
         $magentoRoot = $this->directoryList->getMagentoRoot();
 
-        if (!$this->file->isExists($magentoRoot . '/pub/static.php')) {
-            $this->logger->notice('File "static.php" was not found');
-
-            return;
+        if ($this->file->isExists($magentoRoot . '/pub/front-static.php')) {
+            $this->file->deleteFile($magentoRoot . '/pub/front-static.php');
```

Comment:
`Driver::copy()` overwrites the destination file, so deleting it first shouldn't be necessary